src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/NewCohortWithSingleApprenticeRequest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+// Comment. I think there is a benefit and simplicity in keeping models within Request and Response types as inner classes. It does duplicate the interfaces, 
+// but it makes it clear that the request only requires these explicit fields
+
+namespace SFA.DAS.CommitmentsV2.Api.Types
+{
+    public sealed class NewCohortWithSingleApprenticeRequest
```

Comment:
"NewCohortWithSingleApprenticeRequest" - I saw this more as a "CreateCohortRequest".